examples/mobile/HomeApp/js/clipping-storage.js

```diff
@@ -0,0 +1,24 @@
+class Storage {
+	constructor() {
+		this.activeWebClips = JSON.parse(localStorage.getItem("activeWebClips")) || [];
+	}
+
+	refreshStorage(element, data = []) {
```

Comment:
refreshStorage is never called with second param, what is it for?
why call it?
as I checked in debugger this functions always set empty this[element]

Reply:
For now, this is used only for clear storage (assign empty array to it), but please consider, that changes are everything in this project and this method allows to rewrite whole storage with new data. It can be quite useful. But if you want, I can remove this feature